src/MessageHandler/WebformSubmissionMessageHandler.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\MessageHandler;
+
+use App\Message\NewWebformSubmissionMessage;
+use App\Service\CaseManager;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class WebformSubmissionMessageHandler implements MessageHandlerInterface
+{
+    public function __construct(private CaseManager $caseManager, private HttpClientInterface $client, private $selvbetjeningUserApiToken)
+    {
+    }
+
+    public function __invoke(NewWebformSubmissionMessage $message)
+    {
+        $dataArray = $message->getWebformSubmission();
```

Comment:
Consider renaming this variable to “data” because that's what it is – and we know it's an array.

Reply:
Renamed.